### PR TITLE
chore: linting

### PR DESCRIPTION
**Pull request by mikemckiernan**
Check me...I modified setup.cfg because I was otherwise
receiving the foll error:

```text
merlin/systems/triton/model_config_pb2.py:24: SpecificH ==> specify, specific
```

CI will prolly reveal it soon, but I'll try to fess up before it show it.  I had to use the foll to commit:

```shell
SKIP=interrogate git commit
```

### CODE REVIEW

File: merlin/systems/dag/ensemble.py [hunk 1]
```diff
@@ -21,9 +21,9 @@
 os.environ["PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION"] = "python"
 
 from google.protobuf import text_format  # noqa
+from merlin.dag import Graph  # noqa
```
Comment (benfred) [inline merlin/systems/dag/ensemble.py:24]: I think we should be adding merlin as a known first party package for isort (so it shouldn't be grouped with thirdparty things like google.protobuf).

something like https://github.com/NVIDIA-Merlin/models/pull/216

Comment (benfred) [reply merlin/systems/dag/ensemble.py:24]: (involves adding this line to the isort config https://github.com/NVIDIA-Merlin/models/blob/303a91eebb4859c7d4c5065f345b33e45962c69e/pyproject.toml#L21  )

Comment (karlhigley) [reply merlin/systems/dag/ensemble.py]: Added

**Review by benfred — APPROVED** [review]

**Comment by benfred** [comment]
This change looks good to me - but give me a couple minutes before merging, since I'm curious why the github actions CI didn't kick off here

**Comment by benfred** [comment]
Looks like because of 
```actions/download-artifact@v2, fnkr/github-action-ghr@v1.3, actions/setup-python@v2, actions/checkout@v2, and actions/upload-artifact@v2 are not allowed to be used in NVIDIA-Merlin/systems. Actions in this workflow must be: within a repository owned by NVIDIA-Merlin.
```
- I'm taking a look at fixing

**Pull request merged**